docs/topics/settings.rst

```diff
@@ -1409,7 +1421,9 @@ USER_AGENT
 
 Default: ``"Scrapy/VERSION (+https://scrapy.org)"``
 
-The default User-Agent to use when crawling, unless overridden.
+The default User-Agent to use when crawling, unless overridden. This user agent is
+also used in robots.txt if :setting:`ROBOTSTXT_USER_AGENT` setting is ``None`` and
```

Comment:
Instead of `in robots.txt`, I would say `by RobotsTxtMiddleware`, with a link to the middleware documentation.